keras/backend/tensorflow_backend.py

```diff
@@ -2265,29 +2267,33 @@ def __call__(self, inputs):
             feed_dict[tensor] = value
         session = get_session()
         updated = session.run(self.outputs + [self.updates_op],
-                              feed_dict=feed_dict)
+                              feed_dict=feed_dict,
+                              **self.session_kwargs)
         return updated[:len(self.outputs)]
 
 
-def function(inputs, outputs, updates=None, **kwargs):
+def function(inputs, outputs, updates=None, name=None, **kwargs):
```

Comment:
I did this because `_make_train_function` and `_make_test_function` in `keras/engine/training.py` pass in a `name` kwarg to `K.function`. I've removed the `name` kwarg, but this made the kwargs check in `keras/backend/tensorflow_backend.py:2293` a little messy. Let me know if you have any ideas how this could be written better.